src/components/VDataTable/VCell.js

```diff
@@ -0,0 +1,13 @@
+export default {
+  name: 'v-cell',
```

Comment:
this could cause performance issues. would normally say should be a functional component, but might just need to be a very clean generator function